README.md

```diff
@@ -24,7 +24,7 @@ You can download pre-built binaries at: https://dist.ipfs.io/#ipfs-update
 # You need to CD to a directory _outside_ of your GOPATH.
 $ cd /
 # Install with go modules enabled
-$ GO111MODULE=on go get github.com/ipfs/ipfs-update
+$ GO111MODULE=on go get -u github.com/ipfs/ipfs-update
```

Comment:
The `-u` will cause ipfs-update to break when dependencies update.

Reply:
(without `-u`, we'll use the deps specified by go mod).

Reply:
Interesting, I thought it would only update the source for `ipfs-update` itself (if a previous version already existed on the system). Good to know.
Patch removed from set.

Reply:
Nah. Go mod will always fetch the latest version. That will just cause us to fetch the latest deps as well.